test/test_helper.rb

```diff
@@ -25,4 +25,9 @@ def create_category_and_items(num)
   def teardown
     reset_session!
   end
+
+  def create_user
+    User.create(username: 'ryan',
+                password: 'waffles')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.